R/pkg/R/mllib.R

```diff
@@ -99,10 +114,8 @@ setMethod("spark.glm", signature(data = "SparkDataFrame", formula = "formula"),
             return(new("GeneralizedLinearRegressionModel", jobj = jobj))
           })
 
-#' Fits a generalized linear model (R-compliant).
-#'
-#' Fits a generalized linear model, similarly to R's glm().
-#'
+#' @title Generalized Linear Models (R-compliant)
```

Comment:
I asked @junyangq to try changing `#'` to `#` and see whether it works. We still need a comment for the function but we don't want it to appear on the generated Rd doc.